.circleci/config.yml

```diff
@@ -39,9 +39,9 @@ jobs:
     # 'See docs on artifact collection here https://circleci.com/docs/2.0/artifacts/'
     - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
     # This is based on your 1.0 configuration file or project settings
-    - run:
-        working_directory: ~/orbitdb/ipfs-log
-        command: nvm install 10.15.0 && nvm alias default 10.15.0
+    # - run:
+    #     working_directory: ~/orbitdb/ipfs-log
+    #     command: nvm install 10.15.0 && nvm alias default 10.15.0
```

Comment:
This is a small thing, but something that I think we should do: can we split this particular change into its own PR? It's fairly separate from the core of this PR and would be good to make explicit in the commit history (in master), that the CI config was changed.